project/Settings.scala

```diff
@@ -98,13 +99,16 @@ object Settings {
     dockerUpdateLatest := true,
     dockerCommands ++= List(
       ExecCmd("CMD", "export", "CLASSPATH=lib/*jar")
+//      ExecCmd("CMD", "unmask", "umask g+w") //TODO: doesn't seem to do anything yet
```

Comment:
Based on @calbach's comment in https://broadworkbench.atlassian.net/browse/IA-1086 it seems like the umask needs to be in the `ENTRYPOINT`. I was wondering if another option would be to append it to `~/.bashrc`?

Reply:
Things I also tried for the umask which didn't work for me:

- Append to `~/.bashrc`
- Append to `/etc/profile`
- In the Dockerfile: `umask 0000`